pom.xml

```diff
@@ -30,6 +30,10 @@
             <id>local-file-repo</id>
             <url>file://${basedir}/internal-file-repo</url>
         </repository>
+        <repository>
+            <id>CodeMC</id>
```

Comment:
Why was this added and what does it pull in? Adding maven repos and dependencies is a bit risky, so a small comment would be greatly appreciated :)

Reply:
Bstats deprecated the old maven repository https://bstats.org/getting-started/include-metrics

Reply:
Sounds good :)